components/css/mixins/_fonts.scss

```diff
@@ -0,0 +1,10 @@
+$font-arial:Arial,"Helvetica Neue",Helvetica,sans-serif;
+$font-archivo-narrow:"Archivo Narrow","Arial Narrow",Arial,sans-serif;
+$font-archivo-black:'archivo_blackregular',"Arial Black","Arial Bold",Gadget,sans-serif;
+$font-champion-bantam:"Champion Bantamwt A","Champion Bantamwt B",$font-archivo-narrow;
+$font-champion-feather:"Champion Featherwt A","Champion Featherwt B",$font-archivo-narrow;
+$font-champion-light:"Champion Lightwt A","Champion Lightwt B",$font-archivo-narrow;
+$font-champion-welter:"Champion Welterwt A","Champion Welterwt B",$font-archivo-black;
+$font-champion-middle:"Champion Middlewt A","Champion Middlewt B",$font-archivo-black;
+$font-champion-heavy:"Champion Heavywt A","Champion Heavywt B",$font-archivo-black;
+$font-fontAwesome:FontAwesome;
```

Comment:
Should this really be in the `mixins` folder or should we create a `variables` folder?

Reply:
I would say yes because mixins should contain any SCSS that doesn't directly output CSS. So mixins, variables, etc. If we end up using ITCSS, then we'll have to move it anyways :P.

Reply:
Perhaps before we merge, we should update each use of these fonts in the individual page SCSS files. So anywhere that uses "Arial" will be replaced by the arial font-family variable.